packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -55,18 +54,20 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
    * our account contract.
    * should support the "execFromEntryPoint" and "nonce" methods
    */
-  accountContract?: any // for now
+  accountContract?: SmartAccount_v200
 
   // TODO: both should be V2
 
-  factory?: any // SmartAccountFactory_v100
+  factory?: SmartAccountFactory_v200
 
-  defaultValidationModule: any // for now // BaseValidationModule
-  activeValidationModule: any // for now // BaseValidationModule
+  defaultValidationModule: BaseValidationModule
+  activeValidationModule: BaseValidationModule
 
   constructor(readonly biconomySmartAccountConfig: BiconomySmartAccountV2Config) {
     super(biconomySmartAccountConfig)
-    this.factoryAddress = biconomySmartAccountConfig.factoryAddress
+    // Review: if it's really needed to supply factory address
+    this.factoryAddress =
+      biconomySmartAccountConfig.factoryAddress ?? DEFAULT_BICONOMY_FACTORY_ADDRESS // This would be fetched from V2
```

Comment:
do we plan to call these by backend or we will be using the default ones in sdk from now on?

Reply:
currently I don't see value in fetching it from backend. so it's either provided in config, from constants using provided version or default from constant.